package.json

```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "sinon": "^4.4.8"
+    "sinon": "^4.4.8",
+    "jest": "22.4"
```

Comment:
Jest is actually included already in the 'react' dependencies... as far as I know they come along when `create-react-app` is used and are part of the modules installed when 'react' is already in the package.json file.

Reply:
So am I to:
1) Delete /node_modules
2) Delete ` "jest":  "22.4" ` . from "devDependencies"
3) Leave in the new entry `"jest": { "testEnvironment": "node" }`
For this end result:

```
"devDependencies": {
    "enzyme": "^3.3.0",
    "enzyme-adapter-react-16": "^1.1.1",
    "sinon": "^4.4.8",
    "jest": "22.4"
  },
  "jest": {
    "testEnvironment": "node"
  }

```

Reply:
Close. You'll also want to remove
```
,
  "jest": {
    "testEnvironment": "node"
  }
```

Reply:
Just like it is in the Lambda School repository